.gitignore

```diff
@@ -27,3 +27,6 @@ scripts/__pycache__
 # MacOS
 .DS_Store
 
+# CMake & testing
+/build*
```

Comment:
Why not keep using the existing (already excluded) _build and tmp folders?

Reply:
To make it match the liboqs folder structure, no other reason than that.